examples/react-native/jest.config.js

```diff
@@ -2,7 +2,6 @@ const {resolve} = require('path');
 
 module.exports = {
   preset: 'react-native',
-  testEnvironment: 'jsdom',
   transform: {
     '\\.(js|ts|tsx)$': require.resolve('react-native/jest/preprocessor.js'),
```

Comment:
this is deprecated, but I'm unable to remove it. or rather, removing it works when `jest` is ran from the example folder, but not if run from root. I'm unsure why... will have to solve it when RN 0.69 drops

Reply:
when solved, we can drop the patch we have for the processor as well